### DISCUSSION TITLE
How to ignore element within gallery

### DISCUSSION THREAD

**Discussion by Dan88Man** [dimsemenov/PhotoSwipe#2049] [Q&A]
I would like to know how to ignore i.e. how to exclude an element which is within the "gallery" container. Such an element should be ignored by PhotoSwipe, so that it doesn't get opened by the PhotoSwipe lightbox and when counting the total gallery elements, it shouldn't be considered as part of the gallery. Thanks.

**Comment by dimsemenov** [comment]
```
gallery: '#my-gallery',
children: 'a',
```

`gallery` and `children` options support any CSS selector, so you need to provide a more specific one. For example:
- `children: 'a.my-photo-link'` 
- or `children: 'a:not(.ad-image)'`
